packages/get-credentials/src/index.js

```diff
@@ -0,0 +1,4 @@
+// eslint-disable-next-line import/prefer-default-export
+export { default as getCredentials } from './get-credentials'
```

Comment:
If I do that I get this error:
`TypeError: (0 , _index.getCredentials) is not a function`
😕